File: tourneyAPI/Models/Entities/Characters/Zelda.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Zelda : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Zelda()
    {
        Id = CharacterId.Zelda;
        characterName = CharacterName.ZELDA;
        archetype = Archetype.TURTLE;
        fallSpeed = FallSpeed.FLOATY;
        weightClass = WeightClass.LIGHTWEIGHT;
        tierPlacement = TierPlacement.D;
    }

}

File: tourneyClient/src/models/entities/Characters/Zelda.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Zelda: Character =
{
    id: CharacterId.Zelda,
    characterName: CharacterName.ZELDA,
    archetype: Archetype.TURTLE,
    fallSpeed: FallSpeed.FLOATY,
    weightClass: WeightClass.LIGHTWEIGHT,
    tierPlacement: TierPlacement.D
}

export default Zelda; 